Description: adjust Build.PL to wxwidgets3.0
 allow ^3 as an answer of `wx-config --version' for system wxwidgets
Origin: vendor
Bug: https://rt.cpan.org/Ticket/Display.html?id=94367
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=94367
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-04-01

--- a/Build.PL
+++ b/Build.PL
@@ -153,7 +153,7 @@
     my $wx_config = My::Build::Base->awx_path_search( 'wx-config' );
     if( $wx_config ) {
         my $ans = `wx-config --version`;
-        if( $ans =~ /^2\./ ) {
+        if( $ans =~ /^[23]\./ ) {
             my $prefix = `wx-config --prefix`;
             chomp foreach $ans, $prefix;
             if( _check_installed_widgets_ok($ans, $prefix) ) {
